src/components/Filter/LessonFilter.js

```diff
@@ -4,14 +4,21 @@ import {onFilterCheck, resetFilter} from '../../action_creators';
 import FilterGroup from './FilterGroup';
 import Button from 'react-bootstrap/lib/Button';
 import Panel from 'react-bootstrap/lib/Panel';
+import {getAvailableLessons} from '../../selectors/lesson';
 
 export const LessonFilter = React.createClass({
   render(){
     const filter = this.props.filter || {};
     const filterGroups = Object.keys(filter).map((groupName, idx) => {
       const tagItems = filter[groupName];
       return (
-        <FilterGroup key={idx} groupName={groupName} tagItems={tagItems} onFilterCheck={this.props.onFilterCheck}/>
+        <FilterGroup
+          key={idx}
+          groupName={groupName}
+          availableLessons={this.props.availableLessons}
```

Comment:
Se kommentaren min for FORSLAGET i lesson-selectoren --> Dette er en endring du i såfall må gjøre, dersom du bruker forslaget:
```
          availableLessons={this.props.availableLessons[groupName]}
```